.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+    "arrowParens": "always",
+    "bracketSameLine": false,
+    "bracketSpacing": true,
+    "htmlWhitespaceSensitivity": "css",
```

Comment:
Same with `htmlWhiteSpaceSensitivity`.

Reply:
we do have the top-level html/css files which we likely want to format too.